backend/src/akvo/lumen/lib/invite_impl.clj

```diff
@@ -0,0 +1,85 @@
+(ns akvo.lumen.lib.invite-impl
+  (:require [akvo.lumen.component.keycloak :as keycloak]
+            [akvo.lumen.component.emailer :as emailer]
+            [cheshire.core :as json]
+            [clj-time.coerce :as c]
+            [clj-time.core :as t]
+            [clojure.string :as str]
+            [hugsql.core :as hugsql]
+            [ring.util.response :refer [not-found response redirect]]))
+
+(hugsql/def-db-fns "akvo/lumen/lib/invite.sql")
+
+(defn dev-env? [server-name]
```

Comment:
Why do we need `dev-env?`. Is it just to be able to pprint the full url? Perhaps it would be enough to just print the invite id?.